### ISSUE TITLE
Incompatible with CTM

### ISSUE THREAD

**Issue by eleanoryeung** [LatvianModder/YABBA#114]
Using resource packs with CTM will make all YABBA barrel transparent/invisible.

**Version:**
YABBA 1.1.2.40

**Comment by Straightforthefood** [comment]
Yup, came here to report this. I'm personally using the Unity resource pack.

**Comment by eleanoryeung** [comment]
@Straightforthefood me too :stuck_out_tongue_closed_eyes:

**Comment by deadspidervenom** [comment]
idk if this was fixed or not, but this is the case for me as well

**Comment by Radeka** [comment]
I'm having the same problem, using the current latest versions for both of it, really sad about this since I was mainly installing CTM for Chisel

**Comment by Radeka** [comment]
This was opened in **February** and it's last comment was in **April**. Can someone _please_ address this?

**Comment by LatvianModder** [comment]
*Addressed*. I've been really busy with FTB Mods lately, I haven't had time to look into other mod issues, sorry

**Comment by AdaraChristine** [comment]
Would love to have this fixed so i can update Yabba and Ftblib to current

**Comment by MappaCrappa** [comment]
I'm not using any resource packs, just have CTM installed for Chisel. Would this still be the cause of the invisible barrels I'm experiencing? Is there a workaround for it?

**Comment by deadspidervenom** [comment]
Noticing its been open for over a year going on 2, yea probably.

**Comment by AdaraChristine** [comment]
If there is a work around i would love to know, MV2 is stuck on very old versions of all the mods involved.

**Comment by GabrielSC92** [comment]
Same issue here, would love to be able to use CTM for chisel but this invisible barrel is tilting
